pydis_site/templates/wiki/history.html

```diff
@@ -0,0 +1,155 @@
+{% extends "wiki/article.html" %}
+{% load wiki_tags i18n sekizai_tags static %}
+
+
+{% block wiki_pagetitle %}{% trans "History" %}: {{ article.current_revision.title }}{% endblock %}
+
+{% block wiki_contents_tab %}
+  {% include "wiki/includes/modals.html" %}
+
+  {% addtoblock "js" %}
+    <script type="text/javascript" src="{% static "wiki/js/diffview.js" %}"></script>
+    <script type="text/javascript" src="{% static "wiki/js/diff.js" %}"></script>
+  {% endaddtoblock %}
+  {% addtoblock "css" %}
+    <style type="text/css">
```

Comment:
I'd prefer if this PR didn't have `<style>` tags either, much like how the js was moved into files, I suggest you do the same with the css.

Reply:
Actually, this isn't used, so I'll just remove it.